### ISSUE TITLE
Formatting error on WP_CLI::run_command() handbook page

### ISSUE THREAD

**Issue by terriann** [wp-cli/handbook#185]
Formatting typo causing the last word in the summary to be orphaned in the notes sub section.

![typo-run-command](https://user-images.githubusercontent.com/316632/34398844-287af3a8-eb50-11e7-9a54-53157b7bc413.png)

https://make.wordpress.org/cli/handbook/internal-api/wp-cli-run-command/

**Comment by terriann** [comment]
Just spotted that [this documentation is auto generated from the source code](https://make.wordpress.org/cli/handbook/internal-api/) but it looks like the source code is correct.

https://github.com/wp-cli/wp-cli/blob/master/php/class-wp-cli.php#L1220

My pull request may not be the right way to fix it, please advise?

**Comment by schlessera** [comment]
@terriann First of all, thanks for the issue and pull request.

You are right, though, the pull request cannot be used, as the documentation is being generated from the source files. The generation code is where the bug has to be fixed.

The likely culprit is the `parse_docblock()` method here: https://github.com/wp-cli/handbook/blob/master/bin/command.php#L522-L568

You can see that it splits up the docblock and then guesses the short and long descriptions.

To test the generation, you can use the `wp handbook api-dump` and `wp handbook gen-api-docs` commands while in the root folder of the `wp-cli/handbook` repository.

**Comment by terriann** [comment]
Thanks for the information @schlessera, that was very helpful!

I've submitted a new pull request that would allow for support of short descriptions that are no more than 2 lines and must be followed by a blank line to differentiate between the short and long description.

I tested generation with `wp handbook gen-api-docs` and all the diffs look accurate.

I'd be happy to address any feedback on this change.
